lib/diagnostic.md

```diff
@@ -3,7 +3,10 @@
 Using `inspect element`, write four selectors found on the main page for this GitHub repo.
 
 <!-- your answer starts here -->
-
+$( "ul:eq( 1 )" )
+$( ".repository-content" ).children()
+$( ".file-navigation in-mid-page:lastchild" )
+$( ".repository-content:parent" )
 <!-- your answer ends here -->
```

Comment:
this is a selector: .repository-content

which you wrote, you didn't have to do methods on them.  But good job.  No points off.